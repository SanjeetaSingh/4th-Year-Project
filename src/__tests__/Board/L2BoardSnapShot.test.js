import renderer from 'react-test-renderer'

import Level2 from '../../Level2/L2-Board/Level-2-Board'

it('snapshot test level 2', () => {
    const tree =  renderer.create(<Level2 />).toJSON()
    expect(tree).toMatchSnapshot();
});
