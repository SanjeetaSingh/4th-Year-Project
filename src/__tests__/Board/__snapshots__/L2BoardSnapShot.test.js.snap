// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test level 2 1`] = `
<div
  className="container"
>
  <div
    className="boardcontainer2"
  >
    <div
      className="board2"
    >
      <div
        className="box"
        data-col={0}
        data-row={0}
      >
        <img
          alt="f"
          class="food"
          id="foodOne"
          src="assets/food.png"
        />
      </div>
      <div
        className="box"
        data-col={1}
        data-row={0}
      >
        <img
          alt="c"
          class="cat"
          id="cat"
          src="assets/cat.png"
        />
      </div>
      <div
        className="box"
        data-col={2}
        data-row={0}
      />
      <div
        className="box"
        data-col={3}
        data-row={0}
      >
        <img
          alt="d"
          class="dog"
          id="dog"
          src="assets/dog.png"
        />
      </div>
      <div
        className="box"
        data-col={0}
        data-row={1}
      >
        <img
          alt="h"
          class="hole"
          id="holeOne"
          src="assets/hole.png"
        />
      </div>
      <div
        className="box"
        data-col={1}
        data-row={1}
      />
      <div
        className="box"
        data-col={2}
        data-row={1}
      />
      <div
        className="box"
        data-col={3}
        data-row={1}
      />
      <div
        className="box"
        data-col={0}
        data-row={2}
      />
      <div
        className="box"
        data-col={1}
        data-row={2}
      />
      <div
        className="box"
        data-col={2}
        data-row={2}
      />
      <div
        className="box"
        data-col={3}
        data-row={2}
      />
      <div
        className="box"
        data-col={0}
        data-row={3}
      />
      <div
        className="box"
        data-col={1}
        data-row={3}
      />
      <div
        className="box"
        data-col={2}
        data-row={3}
      />
      <div
        className="box"
        data-col={3}
        data-row={3}
      />
    </div>
  </div>
  <div
    className="bottom-panel"
  >
    <div
      class="border"
    >
      <div
        class="level2Contain"
      >
        <h2>
          Level 2:
        </h2>
        <div
          class="speech"
        >
          Hi there! Your aim for this level is to help the dog get past the obstacles present. You HAVE to use the if statement that will allow you to pass the cat. The cat can only be passed if the dog barks. Use the commands below to create a sequence to get the food bowls. The limit of commands this time is 5. Also be careful of the hole if the dog falls in the hole you will lose
          <br />
           Good luck! 
          <br />
           (To learn more about if statements click the i below)
        </div>
        <div
          class="borderPanel"
        >
          <div
            class="toppart"
          >
            <div
              class="containing"
            >
              <p
                class="number"
                data-testid="counter"
                id="count"
              >
                0/5
              </p>
            </div>
            <p
              class="titles"
            >
              Enter Sequence:
            </p>
            <p
              class="lists"
              id="action"
            />
          </div>
        </div>
        <div
          class="outside"
        >
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Left
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Up
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Down
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Right
          </button>
          <div
            class="buttons-wrapper"
          >
            <button
              class="seemingly-inner-button"
              disabled={false}
              onClick={[Function]}
            >
              <i
                class="fa fa-info"
              />
            </button>
            <button
              class="button"
              disabled={false}
              onClick={[Function]}
            >
              If Statement
            </button>
          </div>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Bark
          </button>
          <button
            class="buttonSub"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
             Submit
          </button>
          <button
            class="buttonClear"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Clear
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
